index.html

```diff
@@ -1,2 +1,193 @@
-<html>
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+	<meta name="viewport" content="width=device-width, initial-scale=1.0">
+	<link rel="stylesheet" type="text/css" href="normalize.css"/>
+	<link rel="stylesheet" type="text/css" href="styles.css"/>
+	<title>Portfolio Project - Steph</title>
+</head>
+
+<body>
+	<nav class="nav-links"> 
+		<div class="links">
+			<a href="#">Home</a>
+			<a href="#aboutme">About Me</a>
+			<a href="#myprojects">My Projects</a>
+			<a href="#contact">Contact</a> 
+		</div>
+	</nav>
+	
+<header>
+	<section class="overall">
+		<div class="header">
```

Comment:
this extra layer of div might not be needed, potential for deletion and the styles to be moved onto the parent